test/scheduler_task_test.rb

```diff
@@ -38,6 +38,20 @@
     Resque::Scheduler.release_master_lock
   end
 
+  test 'can start successfully' do
+    Resque::Scheduler.poll_sleep_amount = nil
```

Comment:
it gets set [in the setup hook](https://github.com/Shopify/resque-scheduler/pull/5/files/1f53e02b84d09821c9ec0f5ba46d5800cfe34815#diff-05b3004fe9e5e23cec630beebd2fe336R8)